pyproject.toml

```diff
@@ -50,9 +50,7 @@ curl-cffi = { version = ">=0.7.2", optional = true }
 docutils = ">=0.21.0"
 eval-type-backport = ">=0.2.0"
 html5lib = { version = ">=1.0", optional = true }
-# TODO: relax the upper bound once the issue is resolved:
-# https://github.com/apify/crawlee-python/issues/774
-httpx = { version = "~=0.27.0", extras = ["brotli", "http2"] }
+httpx = {version = ">=0.28.0", extras = ["brotli", "http2", "zstd"]}
```

Comment:
```suggestion
httpx = {version = ">=0.27.0", extras = ["brotli", "http2", "zstd"]}
```

Reply:
We do not want to limit the compatibility with other versions (without a reason).